rest/user_api_test.go

```diff
@@ -1520,3 +1520,56 @@ func TestPutUserCollectionAccess(t *testing.T) {
 		RequireStatus(t, putResponse, 400)
 	}
 }
+
+func TestUnauthorizedAccessForDB(t *testing.T) {
+	rt := NewRestTester(t, &RestTesterConfig{
+		PersistentConfig: true,
+	})
+	defer rt.Close()
+
+	dbConfig := rt.NewDbConfig()
+
+	dbName := "realdb"
+	rt.CreateDatabase("realdb", dbConfig)
+
+	response := rt.SendRequest(http.MethodGet, "/{{.db}}/", "")
```

Comment:
Does this resolve to "realdb"?  Is there an advantage to using the token here - to me it seems to obscure what's actually being tested.

Reply:
yeah, fixing.